Mathlib/Algebra/Ring/CentroidHom.lean

```diff
@@ -436,6 +439,43 @@ instance : Module R (CentroidHom α) :=
 local notation "L" => AddMonoid.End.mulLeft
 local notation "R" => AddMonoid.End.mulRight
 
+lemma centroid_eq_centralizer_mul_op :
```

Comment:
I think the name of this lemma is a bit weird because `MulOpposite` is not explicitly involved. Maybe this instead?
```suggestion
lemma centroid_eq_centralizer_mulLeftRight :
```

Reply:
`mul_op` was short for "Multiplication Operator".